test/inductor/test_ck_backend.py

```diff
@@ -289,6 +280,8 @@ def addmm(x, a, b, alpha, beta):
             torch.testing.assert_close(Y_compiled, Y_eager)
 
     @unittest.skipIf(not torch.version.hip, "ROCM only")
+    @unittest.skipIf(not PLATFORM_SUPPORTS_BF16, "Scaled mm requires bf16 support")
+    @unittest.skipIf(not PLATFORM_SUPPORTS_FP8, "Scaled mm requires fp8 support")
```

Comment:
Please remove logic in line 293-295 since this is taken care by the skipIfs above. Please also clean up similar logic in the rest of the file.